packages/commonwealth/client/scripts/views/pages/CommunityManagement/CommunityProfile/CommunityProfileForm/CommunityProfileForm.tsx

```diff
@@ -154,9 +159,9 @@ const CommunityProfileForm = () => {
           : DefaultPage.Discussions,
         hasStagesEnabled: community.stagesEnabled,
         customStages:
-          community.customStages === 'true' || !community.customStages
-            ? ''
-            : community.customStages,
+          community.customStages.length > 0
+            ? JSON.stringify(community.customStages)
```

Comment:
i think this is adding `\` in string

<img width="860" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/23c1d718-76ed-44c3-8edc-5bc8bfa43ef1">

Reply:
I haven't been able to replicate this. How did you get to this state? I updated test plan to include `yarn db-all` as there is a migration that is import to run first which may be the cause of your issue.

Reply:
mb, ran the migration script and works.